test/integration/parse_tree_visitor_test.rb

```diff
@@ -0,0 +1,231 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ParseTreeVisitorTest < Minitest::Test
+  include Liquid
+
+  def traversal(template)
```

Comment:
nitpick: I think helpers like this are conventionally put at the end of the test and made `private`